backend/test/modules/doubles-tournament/core/application/CreateTournamentWithTeams.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { testCreateTournamentWithTeamsModule } from './TestCreateTournamentWithTeamsModule';
+import { CreateTournamentWithTeams } from '../../../../../src/modules/doubles-tournament/core/application/command/CreateTournamentWithTeams';
+import { TournamentWithTeamsWasCreated } from '../../../../../src/modules/doubles-tournament/core/domain/event/TournamentWithTeamsWasCreated';
+import { TournamentTeam } from '../../../../../src/modules/doubles-tournament/core/domain/TournamentTeam';
+import { FromListIdGeneratorStub } from '../../../../test-support/shared/core/FromListIdGeneratorStub';
+
+describe('Create Tournament With Teams', () => {
+  it('given only 1 pair of players, when create tournament, then tournament was created with 1 team', async () => {
+    //Given
+    const currentTime = new Date();
+    const entityIdGen = FromListIdGeneratorStub(['TeamId']);
+    const tournamentCreation = testCreateTournamentWithTeamsModule(currentTime, entityIdGen);
```

Comment:
Ten moduł jak widzę, to nazywamy doublesTournament, a nie createTournamentWithTeamsModule :)

Reply:
ajć, robiłem reollbacka i zapomniałem o tym, będzie doublesTournament :P